neuromation/cli/utils.py

```diff
@@ -567,6 +567,9 @@ def pager_maybe(
             click.echo(line)
         return
 
+    # Enforce ANSI sequence handling (colors etc.)
+    os.environ["LESS"] = "-R"
```

Comment:
Maybe set LESS only if it was not set? The user can have custom LESS which includes `-R` or `-r`, but also other options.